frontend/app/routes/_gcweb-app.tsx

```diff
@@ -343,6 +295,26 @@ function Breadcrumbs() {
   );
 }
 
+function MenuOption({ onClick, icon, href, i18nKey }: { onClick: MouseEventHandler<HTMLAnchorElement>; icon?: ReactElement; href: string; i18nKey: string }) {
+  return (
+    <Link className="font-body hover:text-blue-hover ring-blue-hover flex h-[55px] items-center rounded-sm px-4 ring-offset-2 focus:border-none focus:outline-none focus:ring-2" onClick={onClick} to={href} aria-label={i18nKey}>
+      {icon}
+      {i18nKey}
+    </Link>
+  );
+}
+
+function signOutIcon() {
```

Comment:
**Suggestion:** make this a react component and use it as such (ie: `<SignOutIcon />`)